src/Sarif.FunctionalTests/v2/ConverterTestData/FortifyFpr/FortifyTest.fpr.sarif

```diff
@@ -0,0 +1,377 @@
+{
+  "$schema": "http://json.schemastore.org/sarif-2.0.0",
+  "version": "2.0.0",
+  "runs": [
+    {
+      "tool": {
+        "name": "HP Fortify Static Code Analyzer"
+      },
+      "invocations": [
+        {
+          "commandLine": "[REMOVED]insourceanalyzer.exe -b CareerGuideAdminTool -verbose -debug -logfile D:\\AgentWork\\574\\a\\sca_artifacts\\CareerGuideAdminTool_scan.log -scan -f D:\\AgentWork\\574\\a\\sca_artifacts\\CareerGuideAdminTool.fpr",
+          "startTime": "2018-06-14T05:51:41.000Z",
+          "machine": "ACESDLFORT00005",
+          "account": "fortbld",
+          "properties": {
+            "Platform": "Windows Server 2012 R2"
+          }
+        }
+      ],
+      "files": {
+        "ContentFileHandlingController.cs": {
+          "length": 58665,
+          "mimeType": "text/x-csharp",
+          "encoding": "utf-8",
+          "properties": {
+            "FprFileType": "csharp"
```

Comment:
>FprFileType": "csharp [](start = 13, length = 21)

You don't need to persist tool-specific information if it's already represented in a first class SARIF construct. here, we have the relevant mime type, so preserving FPR file type isn't necessary. suggest removing this.